pkg/kfapp/coordinator/coordinator.go

```diff
@@ -173,13 +173,12 @@ func usageReportWarn(components []string) {
 		"To disable it\n" +
 		"If you have already deployed it run the following commands:\n" +
 		"  cd $(pwd)\n" +
-		"  ks delete default -c spartakus\n" +
 		"  kubectl -n ${K8S_NAMESPACE} delete deploy -l app=spartakus\n" +
 		"\n" +
 		"Then run the following command to remove it from your ksonnet app:\n" +
 		"  ks component rm spartakus\n" +
```

Comment:
You are correct we should remove 'ks rm component rm spartakus` because we no longer use ksonnet

Reply:
Thanks, @jlewi , I have removed it.